### ISSUE TITLE
Rule to check for fixtures decorated with usefixtures 

### ISSUE THREAD

**Issue by kasium** [m-burst/flake8-pytest-style#130]
# Rule request

## Description

This rule should check for fixtures which are decorated with ``pytest.mark.usefixtures``. According to the [docs](https://docs.pytest.org/en/6.2.x/reference.html#pytest-mark-usefixtures):

> Also note that this mark has no effect when applied to fixtures.

**Bad example**
```python
@pytest.fixture
def a():
  pass

@pytest.mark.usefixtures("a")
@pytest.fixture
def b():
  pass
```

**Good example**
```python
@pytest.fixture
def a():
  pass

@pytest.fixture
def b(a):
  pass
```

## Rationale

As far as I know, pytest doesn't raise any error or waning if the above pattern is used. It can lead to weird results and broken tests, so a simple check by this plugin could help to prevent such issues.

**Comment by m-burst** [comment]
Hi @kasium,

Thanks for your suggestion!
I've just released v1.5.0 containing this rule under the code `PT025`, along with a new rule `PT026` checking for usage of `pytest.mark.usefixtures` without passing any fixture names.
